spk/tcl/src/installer.sh

```diff
@@ -16,11 +19,24 @@ preinst ()
 postinst ()
 {
 	# Create symlinks
-	ln -s ${SYNOPKG_PKGDEST}/bin/tclsh8.6 /usr/local/bin/tclsh
-	ln -s ${SYNOPKG_PKGDEST}/lib/libtcl8.6.so /usr/local/lib/libtcl.so
-	ln -s ${SYNOPKG_PKGDEST}/lib/libtcl8.6.so /usr/local/lib/libtcl8.6.so
-	ln -s ${SYNOPKG_PKGDEST}/lib/tcl8.6/ /usr/local/lib/tcl8.6
-	export LD_LIBRARY_PATH=/usr/local/lib/
+	mkdir -p /usr/local/bin/ >> ${LOG_FILE} 2>&1
+	mkdir -p /usr/local/lib/ >> ${LOG_FILE} 2>&1
```

Comment:
Please keep your package confined to the package directory. We do not touch system stuff: chances are either that things will be reverted during DSM updates, or you change Synology-controlled files.